annotations/views.py

```diff
@@ -268,11 +268,16 @@ def list_user(request):
     sort_dict = {"user_name":"username", "name":"full_name",
      "aff":"affiliation", "loc":"location"}
 
+    srch_term = request.GET.get('srch-term')
     sort = request.GET.get('sort', 'user_name')
 
     sort_column = sort_dict[sort]
 
-    queryset = VogonUser.objects.exclude(id = -1).order_by(sort_column)
+    if(srch_term):
+        queryset = VogonUser.objects.filter(username__contains = srch_term).exclude(id = -1).order_by(sort_column)
```

Comment:
Might want to use ``__icontains`` rather than ``__contains`` (see [this page](https://docs.djangoproject.com/en/1.9/ref/models/querysets/#icontains)), since mysterious things can happen with capitalization in GET requests...